src/pages/myGamesPage.js

```diff
@@ -61,10 +62,10 @@ export default function MyGames({
 
     return (
         <>
-            <div className="app-body">
+            <div className="app-body" style={{textAlign:"center"}}>
```

Comment:
I haven't seen a ton of inline styling so far in this project, does using some inline styling affect how the CSS classes work?

Reply:
both works, but I'm not using that text align prop in css anymore. I used it there days ago before to start working from App.css

Reply:
Yea we'll want to keep the styling contained to either the app.css file or the chakra tags for readability and ease of editing